src/backend/app/blog.hoon

```diff
@@ -138,7 +138,7 @@
         ::  add html page for this post to
         ::  eyre cache with added mime type
         :*  %pass  /bind  %arvo  %e
-            %set-response  (spat path)
```

Comment:
Too much repetition - if we want to change the prefix to something else, or add more to the prefix, we'll need to change six different locations. Should add a helper function that takes a path and produces a path, e.g.:
```
++  format
  |=  =path
  ^-  ^path
  (spat [~.blog path])
```
`format` isn't the best name, but I can't think of anything better right now - you get the idea.